src/main/java/com/github/monster860/fastdmm/objtree/ObjectTreeParser.java

```diff
@@ -290,7 +291,19 @@ public void doParse(File file, boolean isMainFile) throws IOException
                 if (line.startsWith("#define")) {
                     Matcher m = Pattern.compile("#define +([\\d\\w]+) +(.+)").matcher(line);
                     if (m.find()) {
-                        macros.put(m.group(1), m.group(2).replace("$", "\\$"));
+						String group = m.group(1);
+						if (group.equals("FILE_DIR")) {
+							Matcher quotes = Pattern.compile("^\"(.*)\"$").matcher(m.group(2));
```

Comment:
Oh never mind that's something in FastDMM itself, surprisingly google ended up at that.